tests/typecheck/managers/test_managers.yml

```diff
@@ -394,46 +395,25 @@
         -   path: myapp/models.py
             content: |
                 from django.db import models
-                class MyModelManager(models.Manager):
-
-                    def create(self, **kwargs) -> 'MyModel':
-                          return super().create(**kwargs)
```

Comment:
I think it makes sense to keep this test, but change it to expect error messages to be emitted here